tests/sentry/integrations/github/test_integration.py

```diff
@@ -558,21 +554,65 @@ def test_github_prevent_install_until_pending_deletion_is_complete(self):
         ).exists()
 
     @responses.activate
-    def test_get_trees_for_org(self):
+    def test_derive_code_mappings(self):
         """Fetch the tree representation of a repo"""
         with self.tasks():
             self.assert_setup_flow()
 
         integration = Integration.objects.get(provider=self.provider.key)
         installation = integration.get_installation(self.organization)
         with patch.object(sentry.integrations.github.client.GitHubClientMixin, "page_size", 1):
-            # XXX: Does the installation already have the org stored?
-            installation.get_client().get_trees_for_org(self.organization.slug)
+            trees = installation.get_client().get_trees_for_org(self.organization.slug)
+            # This check is useful since it will be available in the GCP logs
+            assert (
+                self._caplog.records[0].message
+                == "The Github App does not have access to Test-Organization/baz."
+            )
+            assert self._caplog.records[0].levelname == "ERROR"
 
-        # XXX: We need to search filenames in the stack trace
-        # This check is specially useful since it will be available in the GCP logs
-        assert (
-            self._caplog.records[0].message
-            == "The Github App does not have access to Test-Organization/baz."
-        )
-        assert self._caplog.records[0].levelname == "ERROR"
+        expected_code_mappings = [
+            CodeMapping("Test-Organization/foo", "sentry", "src/sentry"),
+            CodeMapping("Test-Organization/foo", "sentry_plugins", "src/sentry_plugins"),
+        ]
+
+        # Case 1 - No matches
```

Comment:
My apologies. I had commented them out in order to fix one of them. They're back again.